### ISSUE TITLE
Option to disable endless scrolling

### ISSUE THREAD

**Issue by yaomtc** [Docile-Alligator/Infinity-For-Reddit#155]
Right now, NoSurf For Reddit is the only open source reddit client with this feature, but NoSurf is too restrictive. Reddit Is Fun is the only other reddit client in general that I could find with this feature, but it's not open source.

**Comment by Docile-Alligator** [comment]
Do you mean you want to manually click a button to load the next page?

**Comment by yaomtc** [comment]
Yeah. That's how the mobile site works (which is now treated as "legacy" and it doesn't seem like it's ever getting fixes) and old.reddit. I don't like an endless, automatic flow of content, I prefer to just read a set portion, which a single page is perfect for.

**Comment by Docile-Alligator** [comment]
This will be difficult to add because the paging library will automatically handle pagination when you scroll down the posts. But I will check if there is an option to change this behavior though.

**Comment by jeremiah-miller** [comment]
Could also just add an option to limit pages to X number of posts, and have it default to however many posts are on a page
